File: android/ac-fluent-ui/src/main/kotlin/com/microsoft/adaptivecards/fluentui/FluentCornerRadii.kt
package com.microsoft.adaptivecards.fluentui

import androidx.compose.runtime.Immutable
import androidx.compose.ui.unit.Dp
import androidx.compose.ui.unit.dp

@Immutable
data class FluentCornerRadii(
    val none: Dp = 0.dp,
    val small: Dp = 2.dp,
    val medium: Dp = 4.dp,
    val large: Dp = 8.dp,
    val xLarge: Dp = 12.dp,
    val circular: Dp = 9999.dp
)

File: ios/Sources/ACFluentUI/FluentCornerRadii.swift
import SwiftUI

public struct FluentCornerRadii: Equatable {
    public var none: CGFloat
    public var small: CGFloat
    public var medium: CGFloat
    public var large: CGFloat
    public var xLarge: CGFloat
    public var circular: CGFloat

    public init(
        none: CGFloat = 0,
        small: CGFloat = 2,
        medium: CGFloat = 4,
        large: CGFloat = 8,
        xLarge: CGFloat = 12,
        circular: CGFloat = 9999
    ) {
        self.none = none
        self.small = small
        self.medium = medium
        self.large = large
        self.xLarge = xLarge
        self.circular = circular
    }
}
